### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by TheRealCL**
navbar + baclkground done + started the cards on the same branch by mistake, will do a new branch for the cards

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,78 @@
...
+        </div>
+        <div class="carousel-item">
+          <img class="d-block w-100" src="https://img.seadn.io/files/d6d897b0d2dd11d03d094adb7bc7f0b6.png?auto=format&fit=max&w=512" alt="Second slide">
+        </div>
+        <div class="carousel-item">
+          <img class="d-block w-100" src="https://img.seadn.io/files/565e3acae9f985f84d461eae15671126.png?fit=max&w=1000" alt="Third slide">
+        </div>
+      </div>
+    </div> %>
+
```
Comment (MBOZAR) [inline app/views/pages/home.html.erb:19]: too many spaces

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,78 @@
...
+  <div class="card-product">
+    <img src="https://img.seadn.io/files/d6d897b0d2dd11d03d094adb7bc7f0b6.png?auto=format&fit=max&w=512" />
+    <div class="card-product-infos">
+      <h3>Art</h3>
+      <h2>Cool Cat #1982</h2>
+      <p>99999 <strong>$NVC</strong></p>
+    </div>
+  </div>
+
+
```
Comment (MBOZAR) [inline app/views/pages/home.html.erb:31]: spaces

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -1,2 +1,78 @@
...
+        <p>99999 <strong>$NVC</strong></p>
+      </div>
+    </div>
+
+  </div>
+
+
+
+
+
```
Comment (MBOZAR) [inline app/views/pages/home.html.erb:76]: spaces

**Review by MBOZAR — APPROVED** [review]

**Review by Rox1610 — APPROVED** [review]
approved with poor style

**Review by Joe-woods — APPROVED** [review]

**Pull request merged**